### DISCUSSION TITLE
Add sql code end of Schema::create

### DISCUSSION THREAD

**Discussion by denizgolbas** [laravel/framework#44398] [Q&A]
I need to add this:

```
partition by range (created_at)

```
to the end of the query that returns Schema::create from migrations.

I tried PostgresGrammar::macro and BluePrint::macro for it. But i couldt do.

Can you help me for how can implement it when create a table?

**Comment by ArdaKaraderi** [comment]
Did you find the solution? We need the same.

**Reply by serious-angel** [reply]
> My teammate solved it with writing package https://github.com/orptech-com/laravel-migration-partition
> 
> [Source](<https://laracasts.com/discuss/channels/laravel/creating-partitioned-table-with-blueprintmacro?reply=836820>)
